### DISCUSSION TITLE
Experiment: Symlinks

### DISCUSSION THREAD

**Discussion by yhakbar** [gruntwork-io/terragrunt#3671] [Experiment Feedback]
Share your feedback on the [symlinks experiment](https://terragrunt.gruntwork.io/docs/reference/experiments/#symlinks).

If you believe that we're ready to end the experiment, and support the feature by default, share that opinion!

**Comment by flah00** [comment]
what version is this available in? i tried `v0.68.7` but only receive errors

```
$ TG_EXPERIMENT=symlinks terragrunt run-all --terragrunt-include-external-dependencies init
08:37:22.736 ERROR  could not find any subfolders with Terragrunt configuration files
08:37:22.736 ERROR  Unable to determine underlying exit code, so Terragrunt will exit with error code 1
```
